code/modules/projectiles/guns/ballistic/toy.dm

```diff
@@ -15,6 +15,18 @@
 	gun_flags = TOY_FIREARM_OVERLAY | NOT_A_REAL_GUN
 	casing_ejector = FALSE
 
+/obj/item/gun/ballistic/automatic/toy/wirecutter_act(mob/living/user, obj/item/I)
```

Comment:
i really suggest moving all this down to /automatic/toy, and just adding a variable/flag that says "can we have our tip removed/do we have a tip", and if its true, do all the stuff, but replace the name/desc setting to be a kinda

```
name = get_untipped_name() //where this proc is on /obj/item/gun/ballistic/automatic/toy
// same for desc
```

Reply:
wait nevermind theres no basetype for toy guns - move it down to item/gun but still check the aformentioned vars/flags

Reply:
i hate you

Reply:
Too niche to be on the base subtype, should be component instead

Reply:
> 

how do I make it so each gun knows which name/desc to pick if all of them are get_untipped_name()?

Reply:
make the component pass through the path of the "non toy" version and it copies vars from that path